com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/ShaderPass/ShaderPassRaytracingForward.hlsl

```diff
@@ -26,9 +26,8 @@ void ClosestHitForward(inout RayIntersection rayIntersection : SV_RayPayload, At
     rayIntersection.t = travelDistance;
     rayIntersection.cone.width += travelDistance * abs(rayIntersection.cone.spreadAngle);
 
-    PositionInputs posInput;
+    PositionInputs posInput = GetPositionInput(rayIntersection.pixelCoord, _ScreenSize.zw);
```

Comment:
@adrien-de-tocqueville is this code expected? and what about the assignment here: posInput.positionWS = fragInput.positionRWS; don't we have a function that do everything?